boards/cuav/nora/nuttx-config/scripts/script.ld

```diff
@@ -186,7 +186,13 @@ SECTIONS
 		*(.gnu.linkonce.d.*)
 		CONSTRUCTORS
 		_edata = ABSOLUTE(.);
-	} > AXI_SRAM AT > FLASH
+
+	    /* Pad out last section as the STM32H7 Flash write size is 256 bits. 32 bytes */
+
+	    . = ALIGN(32);
+		FILL(0xff)
+		. += 32;
+	} > AXI_SRAM AT > FLASH  = 0xff
```

Comment:
According to this [doc](https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_chapter/ld_3.html), the fill should be 2 bytes:
> =fill
    Including =fill in a section definition specifies the initial fill value for that section. You may use any expression to specify fill. Any unallocated holes in the current output section when written to the output file will be filled with the two least significant bytes of the value, repeated as necessary. You can also change the fill value with a FILL statement in the contents of a section definition. 

```suggestion
	} > AXI_SRAM AT > FLASH  = 0xffff
```

Reply:
Yes. Good catch. It was making a the 8bit 0xff into a 16 bit of 0xffff automatically 

but it is worse then that.  Padding is 16 bit. So the size needs to be 16 for 16 bit words not bytes.

Reply:
Master:
```
001de930: 0000 0000 0000 0000 0000 0000 5636 5878  ............V6Xx
001de940: 7800 0000 4816 0024 0100 0000 0000 0000  x...H..$........
001de950: 0000 0000 0000 0000 0000 0000 ff         .............
```

This PR

```
01de960: 0000 0000 0000 0000 0000 0000 5636 5878  ............V6Xx
001de970: 7800 ffff 4816 0024 0100 0000 0000 0000  x...H..$........
001de980: 0000 0000 0000 0000 0000 0000 ffff ffff  ................
001de990: ffff ffff ffff ffff ffff ffff ffff ffff  ................
001de9a0: ffff ffff ffff ffff ffff ffff            ............
```